src/index.js

```diff
@@ -19,6 +19,16 @@ export default class ReactPasswordStrength extends React.Component {
     const { minScore, minLength } = this.props;
     return score >= minScore && length >= minLength;
   }
+  
+  clear() {
+    this.setState({
+      score: 0,
+      isValid: false,
+      password: ''
+    });
+    this.refs['ReactPasswordStrength-input'].value = '';
+    this.forceUpdate();
```

Comment:
We could write this better and avoid using `forceUpdate`. Something like this:

```
  clear() {
    this.setState({
      score: 0,
      isValid: false,
      password: ''
    }, () => {
      this.refs['ReactPasswordStrength-input'].value = '';
    });
  }
```

Also since clearing is technically a change it would be a good idea to call the prop function `changeCallback` in the `setState` callback (see line 43).